applications/nrf_desktop/src/modules/Kconfig.ble_conn_params

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_BLE_CONN_PARAMS_ENABLE
+	bool "BLE connection parameters module"
+	depends on DESKTOP_BT_CENTRAL
+	help
+	  The module updates the Bluetooth connection parameters after the
+	  peripheral discovery, handles connection parameter update requests
+	  from the connected peripherals and increases connection interval for
+	  peripherals while USB is suspended to reduce power consumption.
+
+if DESKTOP_BLE_CONN_PARAMS_ENABLE
+
+config DESKTOP_BLE_USB_MANAGED_CI
+	bool "Manage connection intervals on the USB state change"
+	default y
+	depends on DESKTOP_USB_ENABLE
+	help
+	  Manage BLE connection intervals depending on the USB state. Allows
+	  reduced power consumption of the USB dongle device by setting a larger
+	  connection intervals while the USB enters suspended state. After the
+	  USB resumes normal operation, connection interval is set to the
+	  previous value.
```

Comment:
Why does the `depends on DESKTOP_ROLE_HID_DONGLE` line has been removed? From help it looks as if it stills depends on dongle role.

Reply:
Whole module depends on `DESKTOP_BT_CENTRAL`. I think the new dependency is more appropriate here (the more important fact is that we act as Bluetooth Central and we want to reduce our power consumption while USB is suspended).

Reply:
But can we use it on other devices than dongle?

Reply:
`DESKTOP_BT_CENTRAL` is set only for dongle and has no prompt (so that the module would not be enabled for HID peripheral type anyway)